src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/instance_culling_of_stratholme.cpp

```diff
@@ -79,6 +79,21 @@ enum Yells
     CHROMIE_WHISPER_GUARDIAN_2  = 1,
     CHROMIE_WHISPER_GUARDIAN_3  = 2
 };
+
+enum States
+{
+    WORLDSTATE_SHOW_CRATES = 3479,
+    WORLDSTATE_CRATES_REVEALED = 3480,
+    WORLDSTATE_WAVE_COUNT = 3504,
+    WORLDSTATE_WAVE_MARKER_ES = 3581,
+    WORLDSTATE_WAVE_MARKER_FL = 3582,
+    WORLDSTATE_WAVE_MARKER_KS = 3583,
+    WORLDSTATE_WAVE_MARKER_MR = 3584,
+    WORLDSTATE_WAVE_MARKER_TH = 3585,
+    WORLDSTATE_TIME_GUARDIAN = 3931,
+    WORLDSTATE_TIME_GUARDIAN_SHOW = 3932
```

Comment:
Cosmetic detail: you might want to align the values like you did in `culling_of_stratholme.h`. :-)